apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -124,34 +143,79 @@ Response cachedHttpResponse(String cacheKey) throws IOException {
     private Builder() {
     }
 
+    /**
+     * Set the {@link OkHttpClient} to use for making network requests.
+     *
+     * @param okHttpClient the client to use.
+     * @return The {@link Builder} object to be used for chaining method calls
+     */
     public Builder okHttpClient(@Nonnull OkHttpClient okHttpClient) {
       this.okHttpClient = checkNotNull(okHttpClient, "okHttpClient is null");
       return this;
     }
 
+    /**
+     * <p>Set the API server's base url.</p>
+     *
+     * <b>Base URLs should not have a trailing slash</b>
```

Comment:
I guess it's really depends on server side router configuration, there is no such limitations on client side( on our side). So I would rather to no mention this at all.